OracleWebLogic/dockerfiles/14.1.1.0/Dockerfile.generic-11

```diff
@@ -93,13 +94,14 @@ COPY --from=builder --chown=oracle:oracle /u01 /u01
 
 # Copy scripts
 #-------------
-COPY container-scripts/createAndStartEmptyDomain.sh container-scripts/create-wls-domain.py /u01/oracle/
+COPY container-scripts/createAndStartEmptyDomain.sh container-scripts/create-wls-domain.py container-scripts/get_healthcheck_url.sh /u01/oracle/
 
-RUN chmod +xr $SCRIPT_FILE && \
-    chown oracle:oracle $SCRIPT_FILE /u01/oracle/create-wls-domain.py
+RUN chmod +xr $SCRIPT_FILE $HEALTH_SCRIPT_FILE && \
+    chown oracle:oracle $SCRIPT_FILE /u01/oracle/create-wls-domain.py $HEALTH_SCRIPT_FILE
 
 USER oracle
 
+HEALTHCHECK --start-period=10s --interval=10s --timeout=120s  --retries=12 CMD curl -k -s --fail `$HEALTH_SCRIPT_FILE` || exit 1
```

Comment:
These `HEALTHCHECK` values seem too large. It would take 12x120s (24m) for a service to fail. I would recommend reducing tthe `timeout` to ~5-10s and the `retries` to 3-5.